src/pandas_profiling/model/correlations.py

```diff
@@ -4,13 +4,12 @@
 from contextlib import suppress
 from functools import partial
 from typing import Callable, Dict, List, Optional
-
-import pandas as pd
+import phik
```

Comment:
This import should remain at it's orginal location. It allows users to slim down the dependencies when this coefficient is not used.